test/DynamoCoreWpfTests/WorkspaceGeneralOperations.cs

```diff
@@ -113,7 +113,64 @@ public void VerifyGlobalLacingStrategyUpdatesForLacingStrategies()
             AssertPreviewCount(nodeIds[1].ToString(), 2);
             AssertPreviewCount(nodeIds[2].ToString(), 2);
         }
+        
+        [Test]
+        public void VerifyLacingStrategyForCodeBlockNodes()
+        {
+            var openPath = Path.Combine(TestDirectory, @"core\visualization\LacingStrategyCodeBlockNodes.dyn");
+            ViewModel.OpenCommand.Execute(openPath);
+
+            var workspace = ViewModel.Model.CurrentWorkspace as HomeWorkspaceModel;
+            workspace.RunSettings.RunType = RunType.Automatic;
```

Comment:
Curious why this i needed?

Reply:
> Curious why this i needed?

I just duplicated the existing `VerifyGlobalLacingStrategyUpdatesForLacingStrategies()` test and tweaked it a little so I can't say why it's set up as it is https://github.com/StudioLE/Dynamo/blob/706aaa98488c05abf297891d3cbaeac7e7e89634/test/DynamoCoreWpfTests/WorkspaceGeneralOperations.cs#L57-L63